core/BoidsGenerator.cpp

```diff
@@ -39,5 +59,54 @@ size_t Core::BoidsGenerator::getVerticesBufferSize()
 void Core::BoidsGenerator::updateBoids()
 {
     // Where you need to define your physics function with the three Boids rules;
-    generateBoids();
+    //generateBoids();
+
+    UpdateCoord();
+    WallCyclic();
+    /*m_entities[i].xyz[0]+=m_entities[i].vxyz[0];
+        m_entities[i].xyz[1]+=m_entities[i].vxyz[1];
+        m_entities[i].xyz[2]+=m_entities[i].vxyz[2];*/
+
+    //COLLIDE WALL FUNCTION TO BUILD
+}
+
+float Core::BoidsGenerator::randomZeroOne()
+{
+    return static_cast<float>(rand()) / static_cast<float>(RAND_MAX);
+}
+
+float Core::BoidsGenerator::norm3D(float x, float y, float z)
+{
+    return sqrt(x * x + y * y + z * z);
+}
+
+void Core::BoidsGenerator::UpdateCoord()
+{
+    for (int i = 0; i < m_numEntities; ++i)
+    {
+
+        m_phy_boids[i].xyz[0] += m_phy_boids[i].vxyz[0];
+        m_phy_boids[i].xyz[1] += m_phy_boids[i].vxyz[1];
+        m_phy_boids[i].xyz[2] += m_phy_boids[i].vxyz[2];
+        m_entities[i].xyz = m_phy_boids[i].xyz;
+    }
+}
+
+void Core::BoidsGenerator::WallCyclic()
```

Comment:
Same here, don't be affraid to generate longer function names to make sure they are explicit for everybody involved. Usually it is a good practice to make it sound like an action.